FitnessApp.Web/Controllers/ProductCategoryController.cs

```diff
@@ -115,6 +130,7 @@ public async Task UpdateAsync([FromBody] ProductCategoryDto productCategoryDto)
             _validator.Validate(productCategoryDto, "UpdateProductCategory");
 
             await _productCategoryService.UpdateAsync(productCategoryDto);
+            _eventBus.Publish(new LogEvent(Statuses.Success, "Update", productCategoryDto.GetType().Name.Replace("Dto",""), JsonSerializer.Serialize(productCategoryDto)));
```

Comment:
productCategoryDto.GetType().Name.Replace("Dto","") could you use 100500 symbols and methods instead "productCategory"?

Reply:
I can!